index.html

```diff
@@ -0,0 +1,141 @@
+    </div>
+    <section id="headline">
+
+        <div class="orange-red lato-font">"Hello! Drummers!!"</div>
+        <h1 class="headline-main-text orange-red cocogoose-font">GLOBAL DRUM FESTIVAL & WORKSHOP 2022</h1>
+        <p class="black-like lato-font home-headline-body">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+            Quae, temporibus?
+            Aliquid reiciendis adipisci error voluptatibus illo quos unde, tempore, voluptate porro id velit. Ea
+            voluptatibus maiores nemo!</p>
+        <h3 class="black-like cocogoose-font event-date">15.09.2022(THU) ~ 17(SAT)</h3>
+        <p class="black-like lato-font venue">@ O2 Arena, London, United Kingdom</p>
+    </section>
+    <div id="main-content">
+        <section id="programs" class="programs">
+            <div class="program-header">
+                <h4>Main program</h4>
+                <div class="thin-line"></div>
+            </div>
+            <ul class="program-list">
+                <li>
+                    <i class="solo-icon"></i>
+                    <h5>Drum Solo</h5>
+                    <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Perferendis sapiente vitae nulla
+                        reiciendis.</p>
+                </li>
+                <li>
+                    <i class="shed-icon"></i>
+                    <h5>Shed Session</h5>
+                    <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Perferendis sapiente vitae nulla
+                        reiciendis.</p>
+                </li>
+                <li>
+                    <i class="exhibition-icon"></i>
+                    <h5>Brands Exhibitions</h5>
+                    <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Perferendis sapiente vitae nulla
+                        reiciendis.</p>
+                </li>
+                <li>
+                    <i class="master-class"></i>
+                    <h5>Master Class</h5>
+                    <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Perferendis sapiente vitae nulla
+                        reiciendis.</p>
+                </li>
+                <li>
+                    <i class="Meet-up-icon"></i>
+                    <h5>Meet-up</h5>
+                    <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Perferendis sapiente vitae nulla
+                        reiciendis.</p>
+                </li>
+            </ul>
+
+            <button type="button" class="join-btn">Join Global Drum Festival 2022</button>
+            <div class="program-bg"></div>
+        </section>
+        <section id="performing-artist" class="performing-artists-section">
+            <div class="perform-header">
+                <h4>Performing Drummers</h4>
+                <div class="thin-line">
+                </div>
+            </div>
+            <ul class="performing-artists">
+
+            </ul>
```

Comment:
- [ ] Kindly make sure to follow the design given for this section, below is a comparison between your design with the given design,


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/169385718-ad6a02d4-c76e-4f59-b341-a818d9b1ce30.png) |  ![image](https://user-images.githubusercontent.com/61408860/169385775-550be3dd-6cf7-4a88-8752-b1ae5568e2be.png)

Reply:
- [ ] This issue is well resolved :heavy_check_mark: 
You nailed it my fellow drummer :+1: :heavy_check_mark: